### DISCUSSION TITLE
5 days passed and my payout not received to my account, even it is marked "settled"

### DISCUSSION THREAD

**Discussion by anasfik** [community/community#81941] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,
Thank you for your work firstly.

On my Stripe Connect dashboard, I see that a payout is **settled**, I understand that this takes time to be processed, however, today is the 5th day and I haven't received it in my bank account, I confirmed the input IBAN.

is there any issue, or I just should wait more?

**Comment by DHayes10** [comment, marked as answer]
Hi @anasfik 

We appreciate you reaching out. A payout of د.م.257.04 should now appear on your bank account statement as of 12/22/23, 12:22:17 AM. 

Please let us know if you have any other questions or concerns. 

Thank you, 
GitHub Sponsors

**Reply by mohamedali040118** [reply]
سارت معي عمليت احتيل في ربط عقد وبدي ارد المبلغ حقي

**Reply by Shah11noza** [reply]
Weminedan çikartan paralar çuzdanima gelmedi

**Comment by ravindudil5han** [comment]
> The github said that it will take 16 days, right?

**Comment by Dimension-6320** [comment]
Hiya Geo speaking, I did come across this topic, did you receive your payment? I’m waiting on my yearly payment, first contribution 18-01-2026 but I still haven’t been paid, do I wait until my second payment bill?
